src/MediaPlayer/MediaPlayer.cs

```diff
@@ -319,6 +324,7 @@ public enum MPMediaLibraryAuthorizationStatus : nint {
 
 	[Mac (10,12,2)]
 	[iOS (10,0)]
+	[TV (10,0)]
```

Comment:
So most of these are fields (which I thought should show up) and enum (which I'm not seeing in xtro.

I did find these:

!missing-selector! MPNowPlayingInfoCenter::playbackState not bound
!missing-selector! MPNowPlayingInfoCenter::setPlaybackState: not bound

in tvos.unclassified